scijava/scijava-ops-engine/src/main/java/module-info.java

```diff
@@ -5,42 +5,25 @@
  * corresponding template in templates/ and rerun bin/generate.groovy.
  */
 
-
 	exports org.scijava.ops.engine;
 	exports org.scijava.ops.engine.conversionLoss;
 	exports org.scijava.ops.engine.util;
 
-	opens org.scijava.ops.engine.util.internal to therapi.runtime.javadoc;
-	opens org.scijava.ops.engine to therapi.runtime.javadoc, org.scijava;
-	opens org.scijava.ops.engine.monitor to therapi.runtime.javadoc;
-	opens org.scijava.ops.engine.create to therapi.runtime.javadoc;
-	opens org.scijava.ops.engine.matcher.impl to therapi.runtime.javadoc, org.scijava;
-	opens org.scijava.ops.engine.conversionLoss to therapi.runtime.javadoc;
-	opens org.scijava.ops.engine.copy to therapi.runtime.javadoc;
-	opens org.scijava.ops.engine.log to therapi.runtime.javadoc;
-	opens org.scijava.ops.engine.simplify to therapi.runtime.javadoc;
-	opens org.scijava.ops.engine.impl to therapi.runtime.javadoc, org.scijava;
-	opens org.scijava.ops.engine.yaml to therapi.runtime.javadoc;
-	opens org.scijava.ops.engine.conversionLoss.impl to therapi.runtime.javadoc, org.scijava;
-	opens org.scijava.ops.engine.adapt.complexLift to therapi.runtime.javadoc;
-	opens org.scijava.ops.engine.adapt.lift to therapi.runtime.javadoc;
-	opens org.scijava.ops.engine.struct to therapi.runtime.javadoc;
-	opens org.scijava.ops.engine.adapt.functional to therapi.runtime.javadoc;
-	opens org.scijava.ops.engine.reduce to therapi.runtime.javadoc;
-	opens org.scijava.ops.engine.hint to therapi.runtime.javadoc;
-	opens org.scijava.ops.engine.stats to therapi.runtime.javadoc;
-	opens org.scijava.ops.engine.util to therapi.runtime.javadoc;
-	opens org.scijava.ops.engine.math to therapi.runtime.javadoc;
+	opens org.scijava.ops.engine to org.scijava;
+	opens org.scijava.ops.engine.impl to org.scijava;
+	opens org.scijava.ops.engine.hint to org.scijava.javadoc.parser;
+	opens org.scijava.ops.engine.matcher.impl to org.scijava;
+	opens org.scijava.ops.engine.conversionLoss.impl to org.scijava;
```

Comment:
Maybe file an issue to determine whether these `open`s are actually necessary.

Except for `org.scijava.ops.engine.hint` - why does that need to be opened to the Javadoc parser?

Reply:
No opens anymore!